bgpd/bgp_evpn.c

```diff
@@ -604,8 +604,15 @@ static int bgp_zebra_send_remote_vtep(struct bgp *bgp, struct bgpevpn *vpn,
 /*
  * Build extended communities for EVPN route. RT and ENCAP are
  * applicable to all routes.
+ * TODO: currently kernel doesnt support ipv6 routes with ipv4 nexthops.
+ * This means that we can't do symmetric routing for ipv6 hosts routes
```

Comment:
do you know if IPv4-Mapped IPv6 Addresses can be supported ?

Reply:
These will not be supported either. We have a plan to support them later with general ipv6 route support for EVPN routing.